contracts/Vault.vy

```diff
@@ -1148,6 +1149,7 @@ def addStrategy(
     minDebtPerHarvest: uint256,
     maxDebtPerHarvest: uint256,
     performanceFee: uint256,
+    lossRatioLimit: uint256 = 300 # 3%
```

Comment:
i think the main thing is a deviation both -/+ on the PPS pass a given threshold, there may be need for settings for both loss and profit.

Reply:
Understood, I will make the changes to check price changes in both directions.

Reply:
Maybe change this to `changeLimitRatio`

Reply:
yeah, kind of nitpicky but it's like "change limit" that's a "ratio"